src/Root.js

```diff
@@ -35,6 +36,10 @@ class Root extends React.Component {
     return (
       <ConsortiumManagerContextProvider>
         <Switch>
+          <Route
```

Comment:
it shouldn't be in root, you have ConsortiumManager and PermissionSets for this

Reply:
agree, you could define routes as `<EntrySelector>`'s children.

And as far as you use `<Layer>`  for view, you could provide  `container` or `inRootSet` prop (https://github.com/folio-org/stripes-components/blob/master/lib/Layer/readme.md)